test/fixtures/IL-2022.json

```diff
@@ -0,0 +1,290 @@
+[
+  {
+    "date": "2022-01-17 00:00:00 -0600",
+    "start": "2022-01-16T16:00:00.000Z",
+    "end": "2022-01-17T16:00:00.000Z",
+    "name": "Tu Bishvat (Arbor Day)",
+    "type": "observance",
+    "rule": "15 Shvat",
+    "_weekday": "Mon"
+  },
+  {
+    "date": "2022-02-14 00:00:00 -0600",
+    "start": "2022-02-13T16:00:00.000Z",
```

Comment:
Fast of Esther 2022 is on March 16

https://www.timeanddate.com/holidays/israel/2022